TRN-16/form.php

```diff
@@ -0,0 +1,51 @@
+<?php
+// define variables and set to empty values
+$firstNameErr = $lastNameErr = "";
+$first_name = $last_name =$full_name = "";
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+  if (empty($_POST["firstname"])) {
+    $firstNameErr = "first name is required";
+  } else {
+    $first_name = test_input($_POST["firstname"]);
+    // check if name only contains letters and whitespace
+    if (!preg_match("/^[a-zA-Z]*$/",$first_name)) {
+      $firstNameErr = "Only letters allowed";
+    }
+  }
+
+  if (empty($_POST["lastname"])) {
+    $lastNameErr = "last name is required";
+  } else {
+    $last_name = test_input($_POST["lastname"]);
+    // check if name only contains letters and whitespace
+    if (!preg_match("/^[a-zA-Z]*$/",$last_name)) {
+      $lastNameErr = "Only letters allowed";
+    }
+  }
+  
+  
```

Comment:
remove unnecessary spaces and follow proper indentations

Reply:
Okay!